configure.sh

```diff
@@ -0,0 +1,203 @@
+#!/bin/bash
+
+# Copyright (c) 2018 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#---------------------------------------------------------------------
+# Description: This script is the *ONLY* place where "qemu*" build options
+# should be defined.
+#
+# Note to maintainers:
+#
+# XXX: Every option group *MUST* be documented explaining why it has
+# been specified.
+#---------------------------------------------------------------------
+
+typeset -a qemu_options
+
+qemu_version_file=VERSION
+if [ ! -f ${qemu_version_file} ];then
+	echo "QEMU version file not found"
+	exit 1
+fi
+
+qemu_version_major=$(cat ${qemu_version_file} | cut -d. -f1)
+qemu_version_minor=$(cat ${qemu_version_file} | cut -d. -f2)
+arch=$(arch)
```

Comment:
Please set arch default to x86_64 in case caller does not define it.

Reply:
I'm using `arch` command